app/auth.py

```
@@ -0,0 +1,360 @@
+
+        _, cookie_value = await _create_session(conn, user_id)
+
+    response = RedirectResponse("/gate", status_code=303)
+    _set_session_cookie(response, cookie_value)
+    return response
+
+
+@router.post("/logout")
+async def logout_submit(
+    request: Request,
+    pool: asyncpg.Pool = Depends(get_pool),
```

Ruff (0.15.4)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)